cpp/example/README.md

```diff
@@ -0,0 +1,45 @@
+# Basic Standalone libcudf application
+
+This project contains a simple example demonstrating basic libcudf use case. It
+also show case ways to build your own application based on libcudf in
+minimalistic style.
+
+The example source code loads a csv file that contains stock prices from 4
+companies spanning across 5 days, computes the average of the closing price
+for each company and writes the result in csv format.
+
+## How to compile and execute
+
+The compilation process is automated by a Dockerfile included in the project.
```

Comment:
I don't think our minimal example should *require* docker. Users who want to build using cmake on bare metal should be able to build and run the example. Is docker required for this example? If so, can we break it out so it's optional?

Reply:
My guess is yes. Dockerfile is a pretty good description of the dependency needed to build libcudf. I can report back by trying it out on a bare metal machine (virtual machine).